modules/openapi-generator/src/main/resources/cpp-ue4/model-source.mustache

```diff
@@ -175,9 +175,20 @@ bool {{classname}}::FromJson(const TSharedPtr<FJsonValue>& JsonValue)
 
 	bool ParseSuccess = true;
 
-    {{#vars}}
+	{{#requiredVars}}
+    {{#isWriteOnly}}
+	//Required but write-only. Its absence is reasonable when parsing JSON response.  
+	TryGetJsonValue(*Object, TEXT("{{baseName}}"), {{name}});
+    {{/isWriteOnly}}
+    {{^isWriteOnly}}
+	//Required and not write-only.  Its absence is likely a problem.
```

Comment:
I would rather not have the comments in the generated code to be honest.

Reply:
Noted.  Will change.